Cat2Cat/main.m

```diff
@@ -207,10 +218,34 @@ int main(int argc, const char * argv[])
         [assetCatalogPaths addObjectsFromArray:filesMatchingPattern(url.path)];
     }
     parameters.assetCatalogPaths = assetCatalogPaths;
-    
+
+    parameters.methodNamePrefix = methodNamePrefix;
+
     NSURL *outputUrl = [NSURL URLWithString:[outputDirectory stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding]
                               relativeToURL:basePathUrl];
     parameters.outputDirectory = outputUrl.path;
     
     return parameters;
 }
+
+static BOOL validateMethodNamePrefix(NSString *methodNamePrefix, NSError **error)
+{
+    if ([methodNamePrefix length] < 2) {
+        *error = [NSError errorWithDomain:@"Cat2Cat" code:-1 userInfo:@{ NSLocalizedDescriptionKey : @"Method name prefix must be at least 2 characters." }];
+    } else {
+        NSMutableCharacterSet *charSet = [[NSCharacterSet letterCharacterSet] mutableCopy];
+        [charSet addCharactersInString:@"_"];
+
+        if ([methodNamePrefix rangeOfCharacterFromSet:[charSet invertedSet]].location != NSNotFound) {
+            *error = [NSError errorWithDomain:@"Cat2Cat" code:-1 userInfo:@{ NSLocalizedDescriptionKey : @"Method name prefix contains invalid characters." }];
+        }
+    }
+    return *error == nil;
```

Comment:
Ok, however, that will prevent the app from crashing (in a helpful way) if a developer does not pass in an error pointer, essentially making the error arg optional. Do we want to allow that? It could be valid, since we're also returning false on error from this function, but then we may not have a descriptive error message for the user. Otoh, allowing NULL follows the convention generally used by Apple.

I'm fine either way, just let me know.